src/main/takeDetection/__tests__/takeDetection.test.ts

```diff
@@ -31,127 +37,45 @@ const mockedSentenceSim = getSimilarityScore as jest.Mocked<
   it('should produce expected sentences when given a list of words', () => {
-    const wordList: Word[] = [
-      {
-        word: 'Hello',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'world!',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'Hi',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'there.',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'All',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'ok?',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'test.',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'assessment.',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'exam',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-    ];
+    const wordList = makeWords([
+      'Hello',
```

Comment:
You could just do this as a single string and then `.join(' ')`

Reply:
Did you resolve this without commenting?

Reply:
Sorry, I resolved this together with use `makeBasicWord` one. 
Based on my understanding, `.join(' ')` returns an array as a string, but here we want to pass a list of strings to get a list of words.

Reply:
Ah my bad meant `.split`. Up to you, I think the current approach is clean enough